lib/arc.d.ts

```diff
@@ -267,10 +278,10 @@ declare module "@daostack/arc.js" {
 
   export class ExtendTruffleScheme extends ExtendTruffleContract {
     /**
-     * Returns a string containing an 8-digit hex number whose binary
-     * 1s and 0s represent scheme permissions as follows:
+     * Returns a string containing an 8-digit hex number representing the minimum 
+     * permissions that the scheme may have, as follows:
      *
-     * All 0: Not registered,
+     * All 0: No permissions (note a scheme is always registered when added to a DAO)
```

Comment:
This is related to the permission of schemes at arc.js ..?
There is no such case where the permissions is all 0 on the blockchain.
It is set to one by the controller upon registration.